client/src/app/app.component.html

```diff
@@ -0,0 +1,12 @@
+<div>
+  <mat-toolbar color="primary" class="mat-elevation-z4">
+    {{ appname }}
+  </mat-toolbar>
+  <mat-tab-group mat-align-tabs="start">
+    <mat-tab label="Suche">
+      <lvzviz-search></lvzviz-search>
+    </mat-tab>
+    <mat-tab label="Statistik">
```

Comment:
Is this the tab for the heatmap with timeline feature?

Reply:
Yes, this is the preparation for the heatmap if it continues to be used.